mmv1/third_party/terraform/services/compute/compute_instance_helpers.go.tmpl

```diff
@@ -341,13 +346,16 @@ func expandGracefulShutdownMaxDuration(v interface{}) (*compute.Duration, error)
 
 func flattenScheduling(resp *compute.Scheduling) []map[string]interface{} {
 	schedulingMap := map[string]interface{}{
-		"on_host_maintenance": resp.OnHostMaintenance,
-		"preemptible":         resp.Preemptible,
-		"min_node_cpus":       resp.MinNodeCpus,
-		"provisioning_model":  resp.ProvisioningModel,
+		"on_host_maintenance":         resp.OnHostMaintenance,
+		"preemptible":                 resp.Preemptible,
+		"min_node_cpus":               resp.MinNodeCpus,
+		"provisioning_model":          resp.ProvisioningModel,
 		"instance_termination_action": resp.InstanceTerminationAction,
-		"availability_domain": resp.AvailabilityDomain,
-		"termination_time":    resp.TerminationTime,
+		"availability_domain":         resp.AvailabilityDomain,
+{{ if ne $.TargetVersionName `ga` -}}
```

Comment:
Can we move setting `skip_guest_os_shutdown` down a couple of lines to where `maintenance_interval` and the other beta-only fields are set? It's a minor concern but it keeps the blocks of beta fields together